spec/requests/api/v1/wallets_spec.rb

```diff
@@ -75,23 +72,17 @@
     it 'returns a wallet' do
       get_with_token(
         organization,
-        "/api/v1/wallets/#{wallet.id}"
+        "/api/v1/wallets/#{wallet.id}",
       )
 
       expect(response).to have_http_status(:success)
-
-      result = JSON.parse(response.body, symbolize_names: true)[:wallet]
-
-      expect(result[:lago_id]).to eq(wallet.id)
-      expect(result[:name]).to eq(wallet.name)
+      expect(json[:wallet][:lago_id]).to eq(wallet.id)
+      expect(json[:wallet][:name]).to eq(wallet.name)
     end
 
     context 'when wallet does not exist' do
       it 'returns not found' do
-        get_with_token(
-          organization,
-          "/api/v1/wallets/555"
-        )
+        get_with_token(organization, "/api/v1/wallets/555")
```

Comment:
Style/StringLiterals: Prefer single-quoted strings when you don't need string interpolation or special symbols. (https://rubystyle.guide#consistent-string-literals)